src/App.js

```diff
@@ -1,7 +1,8 @@
 import MainLayout from './Components/MainLayout'
+import { BrowserRouter as Router } from 'react-router-dom'
 
 function App() {
-    return <MainLayout />
+    return <Router><MainLayout/></Router>
```

Comment:
Usually we want the Router, and Routes to be children of the main layout, unless you're planning to have a different layout for each Route. This implies that in the MainLayout component you should render the `children` components. For example:

```javascript
    <MainLayout>
      <Router>
        <Navigation />
        <Switch>
          <Route path="/" exact>
            <Container />
          </Route>
          <Route path="/video">
            <h1>Video</h1>
          </Route>
          <Route path="/login">
            <Login />
          </Route>
        </Switch>
      </Router>
    </MainLayout>
```

NOTE: Notice that the Navigation component is inside the Router, since we will use the links inside it and want to render the specific components